src/services/completions.ts

```diff
@@ -1119,6 +1120,9 @@ namespace ts.Completions {
                                 insertQuestionDot = isRightOfDot && !isRightOfQuestionDot;
                                 type = type.getNonNullableType();
                             }
+                            if (insertQuestionDot && preferences.includeAutomaticOptionalChainCompletions === false) {

```

Comment:
Is this mistake? You don't want to return here.. Instead you want to add this check in `addPropertySymbol` where `insertQuestionDot` is checked

Reply:
Didn't realize it was a closure function and other checks were happening in there. Thanks!

Reply:
Okay, should be fixed now